src/components/createAccount.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react'
+import "../styles/createAccount.css"
+
+export function CreateAccount() {
+  const [firstName, setFirstName] = useState('');
```

Comment:
Ik kan nog niet helemaal ontdekken wat de juiste methode is, 
maar vaak wordt er aangegeven dat bij user input useRef gebruikt moet worden ipv useState. Maar ik kom nog veel tegenstrijdigheden tegen. 
Heb jij bewust voor de useState gekozen hier?

Reply:
Ja met useState wordt per component de status geüpdatet. Bij useRef wordt er geen re-render gedaan.